chain/messagepool/messagepool.go

```diff
@@ -426,6 +426,27 @@ func New(api Provider, ds dtypes.MetadataDS, netName dtypes.NetworkName, j journ
 	return mp, nil
 }
 
+func (mp *MessagePool) ProtectMessages(protect func(cid.Cid) error) error {
```

Comment:
```suggestion
func (mp *MessagePool) ForEachPending(mcid func(cid.Cid) error) error {
```

Or something like that? Basically, the message pool isn't actually protecting _anything_. It's just iterating over its set of messages.